src/components/navigation/AppNavigationBoard.vue

```diff
@@ -133,8 +133,14 @@ export default {
 				actions.push({
 					action: () => {
 						this.hideMenu()
-						// this.boardApi.cloneBoard(this.board)
-						this.$store.dispatch('cloneBoard', this.board)
+						this.loading = true
+						this.$store.dispatch('cloneBoard', this.board).then((newBoard) => {
```

Comment:
please use async await and try, catch :)

Reply:
Sorry, I don't know what this mean. I'm a Newbie. 
Could you please explain?